README.md

```diff
@@ -83,6 +83,7 @@ cmake --build . -j $(sysctl -n hw.ncpuonline)
 ### Installing dependencies on WSL, Debian and Ubuntu
 
 Download and place the 32bit MinGW Development Libraries of [SDL2](https://www.libsdl.org/download-2.0.php), [SDL2_mixer](https://www.libsdl.org/projects/SDL_mixer/), [SDL2_ttf](https://www.libsdl.org/projects/SDL_ttf/) and [Libsodium](https://github.com/jedisct1/libsodium/releases) in `/usr/i686-w64-mingw32`.
+NOTE: SDL2-devel-2.0.9-mingw.tar.gz required
```

Comment:
it should compile fine all the way back to SDL2 2.0.4

Reply:
doesnt compile with SDL2-devel-2.0.12-mingw.tar.gz
it fails at 64%

Reply:
Yeah, I hear that 2.0.12 fails to compile on a lot of platforms, macOS also fails by default. But the text you added is misleading, especially since the issue is with SDL and not us.

Reply:
Maybe rephrase it as 2.0.4-2.0.10 tested/supported, or specifically mention that 2.0.12 has issues.

Reply:
trying to find a way to clarify this since the readme currently only specifies to go download these libraries, by default the latest versions